config/src/main/java/org/springframework/security/config/annotation/method/configuration/EnableMethodSecurity.java

```diff
@@ -97,4 +97,11 @@
 	 */
 	int offset() default 0;
 
+	/**
+	 * Indicate whether to resolve authorization templates with annotation parameters
+	 * @return the resolution policy to use
+	 * @since 6.3
+	 */
+	boolean useAnnotationParameters() default false;
```

Comment:
I don't think that placing the property here makes sense. The property only enables for pre/post but is present on `@EnableMethodSecurity` which can enable for `@Secured` and JSR 250 annotations.

Reply:
I'm not sure that useAnnotationParameters makes sense either. Why is the name `Parameters` used? Perhaps `useTemplates`?

Reply:
I think we can take the annotation setting out for now and leave it as setters for the relevant authorization managers. I'll also update the documentation to reflect that.